articles/00-GettingStarted.md

```diff
@@ -53,11 +71,11 @@ Configuring your scene is extremely simple by simply selecting the following fro
 
 > `Mixed Reality Toolkit -> Configure...`
 
-![Configure](../images/GettingStarted/MixedRealityConfigure.png)
+![Configure](/images/GettingStarted/MixedRealityConfigure.png)
```

Comment:
What's the difference here? The links already work and I'm afraid the actual html docs build will break without this prefix

Reply:
Image linkds fixed